<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detailWindow</class>
 <widget class="QWidget" name="detailWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="detail">
     <column>
      <property name="text">
       <string>条目</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>内容</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="buttonZone" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>12</number>
      </property>
      <item>
       <widget class="QPushButton" name="saveModify">
        <property name="text">
         <string>保存更改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
